src/test/java/tests/LogOutTest.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import pages.HomePage;
+import pages.LoginPage;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class LogOutTest {
+    WebDriver driver = WebDriverInit.getInstance();
+
+    @Test
+    void logOut() {
+        LoginPage loginPage = new LoginPage();
+        loginPage.goToLogInPage();
+        loginPage.logInWithCredentials();
+        HomePage homePage = new HomePage();
+        homePage.clickAvatarButton();
+        homePage.switchToIframe(1);
```

Comment:
move `switchToIframe` and `switchToDefaultContent` methods into` signOut()` so this method will encapsulate all the logic of the output